package.json

```diff
@@ -9,13 +9,13 @@
 	},
 	"scripts": {
 		"commit": "cz",
-		"coverage": "forge coverage --report lcov",
+		"coverage": "forge coverage --report lcov && npm run coverage:filter && npx lcov-summary lcov.info",
+		"coverage:filter": "lcov -r lcov.info 'test/*' 'script/*' 'src/dependencies/*' -o lcov.info",
 		"format": "forge fmt",
 		"format:check": "forge fmt --check",
 		"lint:check": "solhint --config ./.solhint.json 'src/**/*.sol'",
 		"prepare": "husky install",
-		"snapshot": "forge snapshot",
```

Comment:
Did snapshot not provide any meaningful information regarding gas?

Reply:
I placed its output in a diff checker, and saw that its output was the same as what the normal `forge test` command gives, so I deleted it.